workspaces/github-pull-requests-board/plugins/github-pull-requests-board/package.json

```diff
@@ -62,6 +62,8 @@
     "@backstage/frontend-plugin-api": "^0.10.0",
     "@backstage/integration": "^1.16.2",
     "@backstage/plugin-catalog-react": "^1.16.0",
+    "@backstage/plugin-home": "^0.8.7",
```

Comment:
We can remove this line as we don't use anything from this package, there's a comment about this in the `plugin.ts` for full context,